docs/source/user_guide/12_list_of_environment_variables.rst

```diff
@@ -0,0 +1,184 @@
+List of Env vars
+__________________
+
+
+.. list-table:: User List
+    :widths: 35 100 20
+    :header-rows: 1
+
+    * - Environment Variable Name
+      - Description
+      - Default value
+    * - .. envvar:: RADICAL_BASE
+      - root dir for temporary state files,
+      - `$HOME/.radical/`
+    * - .. envvar:: RADICAL_PILOT_DBURL
+      - MongoDB URL string and mandatory for RP to work.
+      - {NOT_SET}
+    * - .. envvar:: RADICAL_UTILS_NTPHOST
+      - NTP host to be used for profile syncing
+      - `0.pool.ntp.org`
+    * - .. envvar:: RADICAL_PILOT_BULK_CB
+      - Enable bulk callbacks for performance boost. This changes the callback signature.
+      - FALSE
+    * - .. envvar:: RADICAL_PILOT_STRICT_CANCEL
+      - Cancel the tasks immediately. #TODO Didn't understood
```

Comment:
#TODO @mtitov Need more details here. You can either comment/or push changes.

Reply:
Limit task cancelation, not to force state "CANCELLED" on a Task Manager side. Note that corresponding command for pilot(s) to cancel tasks is published anyway.

p.s., [code](https://github.com/radical-cybertools/radical.pilot/blob/34db28c651a580276a660b209beed8cfe101bfdc/src/radical/pilot/task_manager.py#L1098-L1113)